### DISCUSSION TITLE
IntroCurso

### DISCUSSION THREAD

**Discussion by TriniGarcP** [LaunchX-InnovaccionVirtual/CursoIntroPython#44] [Q&A] [answered]
Hola, espero puedan ayudarme, estaba realizando el ejercicio de introducción, pero el único que no me salió bien fue del botón(iniciar Launch) en el notebook 
![image](https://user-images.githubusercontent.com/99290564/153268549-90fe2d96-6803-4cfd-a2a5-9e44511b2b71.png)

Sale un error de sintaxis y quisiera saber que hice mal =(
gracias de antemano por su ayuda

**Comment by jdcarrillo123** [comment, marked as answer]
la instrucción else debe de ir a la altura del if

![1](https://user-images.githubusercontent.com/92382017/153272005-b8f37b1a-12d7-46a5-84db-d09b0b32ad77.PNG)

**Reply by TriniGarcP** [reply]
Gracias =) Llegando a mi casa lo voy a checar

**Comment by Karen-Rodriguez** [comment]
Hola, buena noche a todos. Yo pude hacer el ejercicio pero me di cuenta de que tengo estos errores (aunque a mi parecer sí se ejecuta bien). Me pueden decir por qué?

![image](https://user-images.githubusercontent.com/71851591/153325246-98840480-6c06-4234-a3f4-eb89728c9d47.png)

Gracias!!
